lib/_stream_duplex.js

```diff
@@ -71,6 +71,7 @@ function Duplex(options) {
 }
 
 ObjectDefineProperties(Duplex.prototype, {
+  writable: Object.getOwnPropertyDescriptor(Writable.prototype, 'writable'),
```

Comment:
I recommend you run our linter in your IDE. This should use the primordial version:

```suggestion
  writable: ObjectGetOwnPropertyDescriptor(Writable.prototype, 'writable'),
```

Reply:
Fixed.

Hm, yea. I do have eslint plugin installed in VSCode and it works for other projects. However, not for NodeJS unfortunately for some reason.

Reply:
The eslint version that should be used is part of our code base. Therefore the path to eslint itself and also to the rules have to be defined in VSCode for your workspace.

E.g.:

```
  "eslint.nodePath": "/home/ronag/repos/node/tools/",
  "eslint.enable": true,
  "eslint.options": {
      "rulePaths": ["/home/ronag/repos/node/tools/eslint-rules"]
  },
```

Reply:
Oh! Thanks a lot. That makes everything much easier.